servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Single.java

```diff
@@ -1471,9 +1472,10 @@ public final Completable flatMapCompletable(Function<? super T, ? extends Comple
      *      }
      * }</pre>
      *
-     * @param other {@link Single} with which the result of this {@link Single} is to be ambiguated.
+     * @param other {@link Single} to subscribe to and race with this {@link Single} to propagate to the return value.
```

Comment:
Error result will also be propagated. What is effect of cancel?

Reply:
>Creates a new {@link Single} that terminates with the result (either success or error) of either this
     * {@link Single} or the passed {@code other} {@link Single}, whichever terminates first.

I'll add something on cancellation
> After the first source terminates the non-terminated source will be cancelled.